protocols.json

```diff
@@ -24,5 +24,11 @@
   "active": [
     "012-Psithaca",
     "013-PtJakart"
+  ],
+  "sc_rollup": [
+    "alpha"
```

Comment:
Aren't there rollup binaries for `013-PtJakart`?

Reply:
OTOH, [archive with static binaries](https://gitlab.com/tezos/tezos/-/package_files/36986880/download) from upstream release has rollup binaries for 013

Reply:
I'm confused, the static binaries for v13.0 seem to be [here](https://gitlab.com/tezos/tezos/-/packages/6589861) from the link in the [release page](https://gitlab.com/tezos/tezos/-/releases/v13.0) and I don't see rollup binaries there :confused: 

Where is your "archive with static binaries" coming from?

Reply:
If they are undocumented and "unreleased" perhaps we should reconsider releasing them as ourselves as well :unamused:

Reply:
> Where is your "archive with static binaries" coming from?
From [v13.0 release](https://gitlab.com/tezos/tezos/-/releases/v13.0)

Reply:
[as it turns out](https://tezos-dev.slack.com/archives/G01DAPY2Q0N/p1654775720768579) the rollup binaries were not ready for the `v13.0` release, so the ones that are available there should not be considered usable.

@krendelhoff I'm updating the issue and closing this PR.
If, by the time these binaries are ready, this PR is still useful then we can consider reopening it.